pkg/apiserver/authentication/oauth/oauth_options.go

```diff
@@ -141,6 +142,11 @@ type IdentityProviderOptions struct {
 	//  - mixed:  A user entity can be mapped with multiple identifyProvider.
 	MappingMethod MappingMethod `json:"mappingMethod" yaml:"mappingMethod"`
 
+	// ConfirmationNotRequired means that when the user login successfully,
+	// reconfirm the account information is not required.
+	// Username from IDP must math [a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*
+	ConfirmationNotRequired bool `json:"confirmationNotRequired"`
```

Comment:
1. A positive naming convention should be preferred.

Reply:
Compatible with the default configuration of the old version. Confirmation is required by default.